test/hive/compose.yml

```diff
@@ -83,7 +83,7 @@ services:
     # volumes:
     #   - hive-metastore-pgdata:/var/lib/postgresql/data
   presto-coordinator:
-    image: shawnzhu/prestodb:300
+    image: prestosql/presto:latest
```

Comment:
Thanks, but I prefer to stick with `shawnzhu/prestodb` for now because I also use it elsewhere (private project) and want to keep the setup the same. It recently switched to prestosql so should be up to date.